var/spack/repos/builtin/packages/sleef/package.py

```diff
@@ -10,17 +10,20 @@ class Sleef(CMakePackage):
     """SIMD Library for Evaluating Elementary Functions, vectorized libm and DFT."""
 
     homepage = "https://sleef.org"
-    url = "https://github.com/shibatch/sleef/archive/3.2.tar.gz"
+    url = "https://github.com/shibatch/sleef/archive/3.6.tar.gz"
     git = "https://github.com/shibatch/sleef.git"
 
+    maintainers("blapie")
+
     license("BSL-1.0")
 
     version("master", branch="master")
+    version("3.6", commit="a99491afee2bae0b11e9ffbf3211349f43a5fd10", preferred=True)
     version("3.5.1_2020-12-22", commit="e0a003ee838b75d11763aa9c3ef17bf71a725bff")  # py-torch@1.8:
     version(
         "3.5.1",
         sha256="415ee9b1bcc5816989d3d4d92afd0cd3f9ee89cbd5a33eb008e69751e40438ab",
-        preferred=True,
+        deprecated=True,
```

Comment:
Why are we deprecating this version?

Reply:
For no particular reason, I'm happy to remove this line. I was not sure what rule was used to decide whether to deprecate or not.

Reply:
Yeah, please do. Otherwise this LGTM.

Reply:
Done!